app/recordtransfer/static/recordtransfer/css/base/footer.css

```
@@ -1,108 +1,230 @@
+    color: rgba(255, 255, 255, 0.8);
+    text-decoration: none;
 }
 
-.footer-link-title {
-    color: #fcb46c;
-    font-family: Merriweather,serif;
-    font-size: 18pt;
+.footer-more a,
+.footer-more p,
+.footer-contact a,
+.footer-contact p {
+    font-size: small;
 }
 
-.footer-link {
+.footer-contact i,
+.footer-social i {
     color: white;
 }
 
-.footer-link:hover {
-    color: #7a7a7a;
+.contact-content i {
+    margin-right: 10px;
 }
 
-.contact-link-container {
-    display: flex;
-    width: 100%;
-    justify-content: center;
-    align-items: center;
-    color: white;
+.footer a:hover,
+.social-icons i:hover {
+    color: #F5A81C;
+    transition: color 0.3s ease-in-out;
 }
 
-.contact-link-icon {
-    height: 18px;
-    width: 18px;
-    fill: currentColor;
+.footer ul,
+.footer p {
+    margin: 15px 0;
 }
 
-.contact-link-link {
-    text-decoration: none;
-    margin-left: 8px;
-    color: currentColor;
+/* Social icons */
+.social-icons {
+    display: flex;
+    gap: 30px;
 }
 
-.contact-link-container:hover {
-    color: #7a7a7a;
+.social-icon {
+    font-size: 35px;
 }
 
+.hidden-social-content {
+    display: none;
+    margin-top: 30px;
+}
+
+/* Footer bottom */
 .footer-bottom {
-    position: relative;
-    height: 40px;
+    text-align: left;
+    font-size: xx-small;
+    max-width: 1600px;
+    margin: 0 auto;
     width: 100%;
-    background-color: #ab221d;
 }
 
-#copyright {
-    font-size: 0.8rem;
-    color: white;
-    line-height: 40px;
-    text-align: center;
+.footer-bottom p {
+    color: rgba(255, 255, 255, 0.6);
 }
 
-@media(max-width: 700px) {
-    #footer-logo {
-        width: 300px;
+/* Media query for medium screens (two columns) */
+@media (max-width: 1200px) {
+    .footer-more {
+        order: 1;
+    }
+
+    .footer-contact {
+        order: 2;
+    }
+
+    .footer-logo {
+        order: 3;
+    }
+
+    .footer-social {
+        order: 4;
     }
 
     .footer-column {
-        flex: none;
+        width: calc(50% - 30px);
+        margin-bottom: 10px;
+    }
+
+    .footer-logo {
+        display: flex;
+        flex-direction: row;
+        margin-bottom: 10px;
         width: 100%;
-        text-align: center;
+        gap: 0;
+    }
+
+    .nctr-logo-container,
+    .unesco-logo-container {
+        flex: 1;
+        display: flex;
+        justify-content: left;
+        align-items: center;
+    }
+
+    .social-content {
+        display: none;
+    }
+
+    .hidden-social-content {
+        display: block;
+    }
+
+    .logo {
+        max-width: 70%;
+    }
+
+    .social-icon {
+        font-size: 35px !important;
     }
 }
+
+/* Media query for small screens (one column) */
+@media (max-width: 800px) {
+    .footer-more {
+        order: 1;
+    }
+
+    .footer-contact {
+        order: 2;
+    }
+
+    .footer-logo {
+        order: 3;
+    }
+
+    .footer-social {
+        order: 4;
+    }
+
+    .footer-column {
+        width: 100%;
+    }
+
+    .nctr-logo-container,
```

The logos in the footer aren't wrapping when they overwhelm the size of the column (pertains to the *small* screen size). Here's a suggestion to make them break nicely when the screen gets small enough that the images reach their `min-width` of 200px.

```suggestion
    }

    .footer-logo {
        gap: 20px;
        flex-wrap: wrap;
        align-items: center;
        justify-content: center;
    }

    .nctr-logo-container,
```

---

That's a good catch - I've applied your suggestion with the only difference being that I used `row-gap` (for when items are in a column) instead of `gap` to maximize the use of space when the flexbox is a row layout.

Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/525/commits/ea0f738decb30a32abcb5b4e8e2133f978bcc3d7